client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java

```diff
@@ -16,17 +16,7 @@
  */
 package org.apache.rocketmq.client.impl;
 
-import java.io.UnsupportedEncodingException;
-import java.nio.ByteBuffer;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
-import java.util.Set;
-import java.util.concurrent.atomic.AtomicInteger;
+import com.alibaba.fastjson.JSON;
```

Comment:
Import codestyle.xml in your IDE first
https://rocketmq.apache.org/docs/code-guidelines/